packages/bits/angular.json

```diff
@@ -310,5 +311,8 @@
       "prefix": "nui",
       "lintFix": true
     }
+  },
+  "cli": {
+    "analytics": false
```

Comment:
I forget what this is for.

Reply:
This stops angular from asking you everytime you run the app if you want to send analytics back to to them. I can remove it if you like though

Reply:
seems fine to keep it :)